InterviewFlashCards/MenuViewController.swift

```diff
@@ -9,19 +9,40 @@
 import UIKit
 
 class MenuViewController: UIViewController {
-    
+
+    enum Section: Int {
+        case ios
+        case dataStructures
+        case algorithms
+    }
```

Comment:
`Int` based enum confined to the VC. No reason that it should be an Int based enum else where. The section is mapped to a `String` based enum later.